kube/autoscale/hpa-ima.yml

```diff
@@ -19,7 +19,7 @@ spec:
   maxReplicas: 5
   {{ else }}
   minReplicas: 1
-  maxReplicas: 5
```

Comment:
are we sure we don't want to scale up to 5 incase of a high demand? 2 would usually be the minimum

Reply:
we usually do 3 @JHoldergov

Reply:
right, so @Rhodine-orleans-lindsay can scale that to 3 then?

Reply:
branch takes up a lot of resources, can we not have one? in prod yeah have 3

Reply:
usually in a file we have kube/non-prod/hpa.yml and then kube/non-prod/hpa.yml if we are sure we have split the 2 then that is good but if not we may have to separate the two

Reply:
usually in a file we have kube/non-prod/hpa.yml and then kube/non-prod/hpa.yml if we are sure we have split the 2 then that is good but if not we may have to separate the two

Reply:
@sulthan-ahmed @Rhodine-orleans-lindsay ^^

Reply:
@JHoldergov there are if/else statements in the file e.g.` {{ if or (eq .KUBE_NAMESPACE .STG_ENV) (eq .KUBE_NAMESPACE .PROD_ENV) }`} scales the pods for the staging and prod environments